_episodes/02-r-dataframes.md

```diff
@@ -106,26 +108,27 @@ You may have noticed another unfamiliar operator above, the double colon (`::`).
 Let's explore a few other examples of how we can use dplyr and pipes to manipulate our dataframe.
 
 ~~~
-proj58_matched_2016 %>% slice(1:5) #selects rows 1 to 5 in the dplyr way
-# Take proj58_matched_2016 AND THEN slice rows 1 through 5.
+cbcnr_matched_2016 %>% slice(1:5) #selects rows 1 to 5 in the dplyr way
+# Take cbcnr_matched_2016 AND THEN slice rows 1 through 5.
 
 #We can also use multiple pipes.
-proj58_matched_2016 %>%
-  distinct(detectedby) %>% nrow #number of arrays that detected my fish in dplyr!
-# Take proj58_matched_2016 AND THEN select only the unique entries in the detectedby column AND THEN count them with nrow.
+cbcnr_matched_2016 %>% 
+  distinct(detectedBy) %>% 
+  nrow #number of arrays that detected my fish in dplyr!
+# Take cbcnr_matched_2016 AND THEN select only the unique entries in the detectedBy column AND THEN count them with nrow.
 
 #We can do the same as above with other columns too.
-proj58_matched_2016 %>%
-  distinct(catalognumber) %>%
-  nrow #number of animals that were detected
-# Take proj58_matched_2016 AND THEN select only the unique entries in the catalognumber column AND THEN count them with nrow.
+cbcnr_matched_2016 %>% 
+  distinct(catalogNumber) %>% 
+  nrow #number of animals that were detected 
+# Take cbcnr_matched_2016 AND THEN select only the unique entries in the catalogNumber column AND THEN count them with nrow.
 
 #We can use filtering to conditionally select rows as well.
-proj58_matched_2016 %>% filter(catalognumber=="PROJ58-1191602-2014-07-24")
-# Take proj58_matched_2016 AND THEN select only those rows where catalognumber is equal to the above value.
+cbcnr_matched_2016 %>% filter(catalogNumber=="CBCNR-1191602-2014-07-24") 
+# Take cbcnr_matched_2016 AND THEN select only those rows where catalogNumber is equal to the above value.
 
-proj58_matched_2016 %>% filter(monthcollected >= 10) #all dets in/after October of 2016
-# Take proj58_matched_2016 AND THEN select only those rows where monthcollected is greater than or equal to 10.
+cbcnr_matched_2016 %>% filter(decimalLatitude >= 38)  #all dets in/after October of 2016
```

Comment:
comment is not updated but code is fine.

Reply:
not seeing this in the file - the comment was updated to "# Take cbcnr_matched_2016 AND THEN select only those rows where latitude is greater than or equal to 38. "